README.md

```diff
@@ -46,9 +46,10 @@ A curated list of awesome companies that offer their tools and services for free
 - [AppVeyor](https://www.appveyor.com/) - CI/CD for Windows developers.
 - [CircleCI](https://circleci.com/) - Docker-based building with support of customized workflows.
 - [Codefresh](https://codefresh.io/) - Docker-native CI/CD.
-- [Codeship](https://codeship.com/) - Continuous integration, delivery, and deployment.
+- [CloudBees CodeShip](https://codeship.com/) - Continuous integration, delivery, and deployment.
 - [continuousphp](https://continuousphp.com/) - CI/CD for PHP applications.
 - [DeployHQ](https://www.deployhq.com/) `requires-approval` - Deployment automation.
+- [Jenkins X](https://jenkins-x.io/) - CI/CD for Kubernetes.
```

Comment:
I can't tell from their website, is Jenkins X a commercial product?

Reply:
It's open source, see here: https://github.com/jenkins-x/jx/blob/master/LICENSE